libraries/cli/lib/bin.ts

```diff
@@ -139,6 +137,11 @@ async function main() {
     .version(versions)
     .showHidden(false)
     .demandCommand()
+    .env("SYNTEST")
+    .middleware(async (argv) => {
+      // Set the arguments in the module manager
+      ModuleManager.instance.args = argv;
```

Comment:
Why are these args needed in the module manager?

Reply:
From the module manager, I pass them to all the modules.
It is the only way to enable plugins to use the arguments.